README.md

```diff
@@ -88,6 +88,7 @@ Installing via the pip command will install the following packages:
 * [scikit-learn>=0.22.0](http://scikit-learn.org/stable/install.html#)
 
 ```
+> pip install indelpost --no-binary indelpost --no-build-isolation  
```

Comment:
Isn't indelpost a requirement that should be installed automatically by pip when installing RNAIndel?

Reply:
indelPost needs to be built using pysam installed on the user's local. The current setup.py does use the indelpost binary uploaded to PyPI, which can cause a binary incompatibility. I think, for now, manually installing indelPost is safer.